lms/validation/_base.py

```diff
@@ -136,14 +137,18 @@ def parse(self, *args, **kwargs):
         either return the successfully parsed params or raise a validation
         error.
 
-        :raise lms.validation.ValidationError: if the response isn't valid
+        :raise ExternalRequestError: if the response isn't valid
         """
         try:
-            result = self.load(self.context["response"], *args, **kwargs)
+            return self.load(self.context["response"], *args, **kwargs)
         except marshmallow.ValidationError as err:
-            raise ValidationError(messages=err.messages) from err
+            response = self.context.get("response")
+            request = response.request if response is not None else None
+            validation_errors = err.messages
```

Comment:
I'm not sure if it's strictly necessary but I'm being careful to make sure this code doesn't crash if `response` is None, hence the `request = response.request if response is not None else None`. Some tests fail if you don't do this. I suspect this may be the fault of the tests rather than an actual bug in the production code. But there's no harm in being careful here I guess